src/components/MoneyRequestReportView/MoneyRequestReportActionsList.tsx

```diff
@@ -115,6 +112,18 @@ function MoneyRequestReportActionsList({report, reportActions = [], transactions
         return filteredActions.toReversed();
     }, [reportActions, isOffline, canPerformWriteAction]);
 
+    const reportActionSize = useRef(visibleReportActions.length);
+    const lastAction = visibleReportActions.at(-1);
+    const lastActionIndex = lastAction?.reportActionID;
+    const previousLastIndex = useRef(lastActionIndex);
```

Comment:
There is `usePrevious` hook

Reply:
thanks, I agree with you, but at this point I'd prefer to keep code the same both in`MoneyRequestReportActionsList` and standard `ReportActionsList`.

I will add this comment to another issue where I will be de-duplicating common code and fix it there.